### PR TITLE
Add Games Tests with Mock (#57)

### PR DESCRIPTION
**Pull request by mikesmith**
✅ Add Games endpoint tests using Mock
✅ Update travis.yml to run games tests

One test may fail due to (#65). Once that is fixed, the tests should run clean.

Please let me know what I can update/fix. 

If approved, this may close (#57)

Thanks!

### CODE REVIEW

File: backend/games/tests.py [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+                "id": 2155,
+                "cover": {"id": 43486, "image_id": "e57qvevkjfapzizl3qhn"},
+                "first_release_date": 1316649600,
+                "name": "Dark Souls",
+                "slug": "dark-souls",
+            }]
+        mock_response.json.return_value = expected_dict
+        mock_post.return_value = mock_response
+
+        """Ensure we can obtain a game from IGDB."""
```
Comment (danielgrijalva) [inline backend/games/tests.py]: Can you put the docstrings just below the function definition? :sweat_smile: 
Like this:
```python
def test_get_game(self, mock_post):
    """Ensure we can obtain a game from IGDB."""
```

**Comment by danielgrijalva** [comment]
Indeed, the build is failing because of what you said :thinking: 

But this looks really good! Thanks a lot, I couldn't figure out how to use Mock API :sweat_smile: I can merge once we solve #65 :+1:

**Comment by mikesmith** [comment]
Great! Thanks for the feedback. Will make the changes and hopefully solve #65 as well.

**Comment by mikesmith** [comment]
Whoops. I think I messed up when I tried re-syncing my local branch with the latest changes and with my remote branch.

File: backend/games/tests.py (hunk 1 above)
Comment (mikesmith) [reply backend/games/tests.py]: Should be fixed now. Thanks!

**Comment by danielgrijalva** [comment]
No worries - looks good! Merging now.

**Pull request merged**